course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+- Check up on the collaborative process
+    - An informal discussion that doubles as a chance to flesh out where the project is going. How is the collaborative process working? Do we need to make adjustments to get the project done? Use our answers to these questions to help populate/adjust the github issue list.
+- Looking towards the future
+    - Use this discussion as an opportunity to get feedback on the course, and to help guide students in embracing open science and reproducible research in their future endeavours. 
+
+
+### Exercises
+These are largely covered by the topics in the lectures, but if you see something that is not and that you think is important, comment on that line suggesting where to fit it in.
+
+- Find university IP policy, and discuss
+- Sign up for github
+- Brainstorm questions for project data set
+    - What you'd like to ask of your project data set? What questions can be answered with the data set directly? What questions can be answered with additional contextual data? Ask a variety of questions. Write down questions and hypotheses in your blog/etherpad so you have them on record.
+- Selecting collaborative writing tools
+    - Search the web with your group for collaborative writing tools and platforms. Some places to start: Overleaf, Authorea, Google Docs...even a Word file in a shared Dropbox… really, anything that would allow multiple people to have edit access to a document.
+- DATA-README for the project
+- Authorship exercise
+    - It's always a good idea to set authorship expectations early in the game- it helps contributors understand their respective roles. Discuss with your group- who needs to be on the author list beyond the group members? If people outside the group are to be included, how will they know what their respective roles are? Author order can be determined at a later date, but begin thinking about this as well!
+- Reading
+    - To prep for next week’s activities, please skim through The Quartz guide to bad data.
+- Personas
+    - Do a “Persona” exercise designed to help understand goals and motivations of potential data donors. The persona is an imaginary data producer, and our goal is to get that person to share their data. Based on real-world observations and understandings of actual potential or current donors, sketch out this persona, and identify his or her motivations. The persona is used in business and software development to help designers understand and empathize with their users.
+- Setup R-studio with Github
+- Loops, Conditionals and functions
```

Comment:
As I mentioned previously, loops should be discouraged here. R is not like other programming languages this way. There are a ton of base functionality in R that uses vectorization and/or the tidyverse-style of doing things (focus on data.frames, mapping functions on the entire data.frame or subsets of it rather than looping).